android/app/src/main/java/com/nt4f04und/sweyer/channels/ContentChannel.java

```diff
@@ -233,10 +233,10 @@ public void onMethodCall(MethodCall call, @NotNull MethodChannel.Result result)
                if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.R) {
                   this.result = result;
                   Boolean value = call.argument("value");
-                  ArrayList<HashMap<String, Object>> songs = call.argument("songs");
+                  ArrayList<ArrayList<Object>> songsIds = call.argument("songsIds");
                   ArrayList<Uri> uris = new ArrayList<>();
-                  for (HashMap<String, Object> song : songs) {
-                     Long id = GeneralHandler.getLong(song.get("id"));
+                  for (Object songId : songsIds) {
```

Comment:
I don't see the lint, also I believe that null we lead to crash here. Am I missing something?

Reply:
> I believe that nulls would lead to crash here.

Exactly, it would crash on the Java side. Wouldn't it be better to return an error to Flutter, so we can display an error toast or something?

Reply:
Firstly, the exception is catched is reported the Flutter anyways, secondly, if a crash like this happens, this means that ContentChannel on Dart side somehow has a wrong typing.